content/en/data_security/agent.md

```diff
@@ -88,6 +88,74 @@ If you have a requirement to avoid storing secrets in plaintext in the Agent's c
 
 For more information, see the [Secrets Management][18] documentation.
 
+## Telemetry collection
+
+{{< site-region region="gov" >}}
+
+Datadog on non-government sites will collect environmental, performance, and feature usage information about the Datadog Agent. When Agent detects a government site or FIPS usage Agent will automatically disable this telemetry collection. When such detection is impossible (for example when proxy is used) Agent telemetry will be emitted but immediately dropped at intake. To avoid this emission in the first place we recommend to disable Agent telemetry explicitly by updating the `agent_telemetry` setting in the Agent configuration file, as shown in the example below.
+{{< tabs >}}
+{{% tab "datadog.yaml" %}}
+
+```yaml
+agent_telemetry:
+  enabled: false
+```
+{{% /tab %}}
+{{< /tabs >}}
+{{< /site-region >}}
+{{< site-region region="us,us3,us5,eu,ap1" >}}
+Datadog may collect environmental, performance, and feature usage information about the Datadog Agent. This may include diagnostic logs and crash dumps of the Datadog Agent with obfuscated stack traces to support and further improve the Datadog Agent.
+
+You can disable this telemetry collection by updating the `agent_telemetry` setting in the Agent configuration file, as shown in the example below.
+{{< tabs >}}
+{{% tab "datadog.yaml" %}}
+
+```yaml
+agent_telemetry:
+  enabled: false
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+**Telemetry content:**
```

Comment:
As discussed live, please add a one-line description about what each metadata and metric represents

Reply:
in process